projects/1.16/assets/ravage-and-cabbage/ravageandcabbage/lang/zh_cn.json

```
@@ -0,0 +1,24 @@
+{
+	"itemGroup.ravageandcabbage_item_group": "Ravage & Cabbage",
+	
+	"item.ravageandcabbage.cabbage": "卷心菜",
+	"item.ravageandcabbage.cabbage_seeds": "卷心菜种子",
+	"item.ravageandcabbage.ravager_milk": "劫掠兽奶桶",
+	"item.ravageandcabbage.leather_horn_armor": "皮革护角",
+	"item.ravageandcabbage.golden_horn_armor": "金护角",
+	"item.ravageandcabbage.iron_horn_armor": "铁护角",
+	"item.ravageandcabbage.diamond_horn_armor": "钻石护角",
+	"item.ravageandcabbage.netherite_horn_armor": "下界合金护角",
+
+	"item.ravageandcabbage.cabbager_spawn_egg": "卷心菜农刷怪蛋",
+	"item.ravageandcabbage.ravager_spawn_egg": "劫掠兽刷怪蛋",
+
+	"entity.ravageandcabbage.ravager": "劫掠兽",
+	"entity.ravageandcabbage.cabbager": "卷心菜农",
+	"entity.ravageandcabbage.cabbage": "被投掷的卷心菜",

```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
	"entity.ravageandcabbage.cabbage": "扔出的卷心菜",
```
原文：**Thrown Cabbage**